osv/repos.py

```diff
@@ -107,15 +107,17 @@ def clone(git_url, checkout_dir, git_callbacks=None):
 def clone_with_retries(git_url, checkout_dir, git_callbacks=None, branch=None):
   """Clone with retries."""
   logging.info('Cloning %s to %s', git_url, checkout_dir)
-  for _ in range(CLONE_TRIES):
+  for attempt in range(CLONE_TRIES):
     try:
       repo = clone(git_url, checkout_dir, git_callbacks)
       repo.cache = {}
       if branch:
         _checkout_branch(repo, branch)
       return repo
     except (pygit2.GitError, subprocess.CalledProcessError) as e:
-      logging.error('Clone failed: %s', str(e))
+      if attempt == range(CLONE_TRIES)[-1]:
+        logging.error('Clone failed on attempt %d of %d: %s', attempt + 1,
+                      CLONE_TRIES, str(e))
```

Comment:
`attempt + 1` will always equal `CLONE_TRIES` here.
```suggestion
        logging.error('Clone failed after %d attempts: %s', CLONE_TRIES,
                      str(e))
```
Maybe?